README.md

```diff
@@ -46,11 +46,11 @@ The code present in this repository is used to define and deploy a dockerised co
 This is done by calling a [module](https://github.com/companieshouse/terraform-modules/tree/main/aws/ecs) from terraform-modules. Application specific attributes are injected and the service is then deployed using Terraform via the CICD platform 'Concourse'.
 
 
-Application specific attributes | Value                                | Description
-:---------|:-----------------------------------------------------------------------------|:-----------
-**ECS Cluster**        | company-requests                                  | ECS cluster (stack) the service belongs to
-**Load balancer**      | {env}-chs-chgovuk                                | The load balancer that sits in front of the service
-**Concourse pipeline**     |[Pipeline link](https://ci-platform.companieshouse.gov.uk/teams/team-development/pipelines/lfp-pay-web) <br> [Pipeline code](https://github.com/companieshouse/ci-pipelines/blob/master/pipelines/ssplatform/team-development/lfp-pay-web)                                  | Concourse pipeline link in shared services
+Application specific attributes | Value                                                                                                                                                                                                                                      | Description
+:---------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------
+**ECS Cluster**        | company-requests                                                                                                                                                                                                                           | ECS cluster (stack) the service belongs to
+**Load balancer**      | {env}-chs-chgovuk                                                                                                                                                                                                                          | The load balancer that sits in front of the service
+**Concourse pipeline**     | [Pipeline link](https://ci-platform.companieshouse.gov.uk/teams/team-development/pipelines/penalty-payment-web) <br> [Pipeline code](https://github.com/companieshouse/ci-pipelines/blob/master/pipelines/ssplatform/team-development/penalty-payment-web) | Concourse pipeline link in shared services
```

Comment:
Note -- This is dependant on https://github.com/companieshouse/ci-pipelines/pull/3793/files being pushed, renaming the pipeline names, etc.
Might be worth keeping this LFP for now, until we get the pipeline changes approved -- there's a cleanup task after that for pay-api too, so we can create new ticket for this additional config refactoring.

Reply:
Reverted back to lfp-pay-web as suggested